front-end/src/Profile.js

```diff
@@ -82,7 +106,32 @@ const UserInfo = ({ data }) => {
 		</Link>
 	    </div>
 	    <div>
-		<button className="profButton">Settings</button>
+		<button className="profButton" onClick={toggleModal}>Settings</button>
```

Comment:
Looks very good, thank you Lev! A very minor and nitpicky thing--I think the standard for class names for HTML/CSS is `class-name`, with all lowercase and dashes. I'd personally prefer it if you could change the class names to fit this format, but it's certainly not necessary.

Reply:
Changed.